build_tools/third_party/sdl2/BUILD.overlay

```diff
@@ -0,0 +1,168 @@
+            "src/video/qnx/**",
+            "src/core/linux/**",
+            "src/thread/**",
+            "src/core/unix/**",
+        ],
+    ) + glob([
+        "src/thread/*",
+        "src/thread/windows/*",
+        "src/thread/generic/SDL_syscond.*",
+    ]),
+    "//conditions:default": glob(
+        include = [
+            "src/**/*.c",
+            "src/**/*.h",
+        ],
+        exclude = SDL_DEFAULT_SRC_EXCLUDES,
+    ),
+})
+
+sdl_headers = [
+    "include/begin_code.h",
+    "include/close_code.h",
+    "include/SDL.h",
+    "include/SDL_assert.h",
+    "include/SDL_atomic.h",
+    "include/SDL_audio.h",
+    "include/SDL_bits.h",
+    "include/SDL_blendmode.h",
+    "include/SDL_clipboard.h",
+    "include/SDL_config_minimal.h",
+    "include/SDL_cpuinfo.h",
+    "include/SDL_egl.h",
+    "include/SDL_endian.h",
+    "include/SDL_error.h",
+    "include/SDL_events.h",
+    "include/SDL_filesystem.h",
+    "include/SDL_gamecontroller.h",
+    "include/SDL_gesture.h",
+    "include/SDL_haptic.h",
+    "include/SDL_hints.h",
+    "include/SDL_joystick.h",
+    "include/SDL_keyboard.h",
+    "include/SDL_keycode.h",
+    "include/SDL_loadso.h",
+    "include/SDL_log.h",
+    "include/SDL_main.h",
+    "include/SDL_messagebox.h",
+    "include/SDL_metal.h",
+    "include/SDL_mouse.h",
+    "include/SDL_mutex.h",
+    "include/SDL_name.h",
+    "include/SDL_opengl.h",
+    "include/SDL_opengl_glext.h",
+    "include/SDL_opengles.h",
+    "include/SDL_opengles2.h",
+    "include/SDL_pixels.h",
+    "include/SDL_platform.h",
+    "include/SDL_power.h",
+    "include/SDL_quit.h",
+    "include/SDL_rect.h",
+    "include/SDL_render.h",
+    "include/SDL_revision.h",
+    "include/SDL_rwops.h",
+    "include/SDL_scancode.h",
+    "include/SDL_sensor.h",
+    "include/SDL_shape.h",
+    "include/SDL_stdinc.h",
+    "include/SDL_surface.h",
+    "include/SDL_system.h",
+    "include/SDL_syswm.h",
+    "include/SDL_thread.h",
+    "include/SDL_timer.h",
+    "include/SDL_touch.h",
+    "include/SDL_version.h",
+    "include/SDL_video.h",
+    "include/SDL_vulkan.h",
+] + select({
+    "@bazel_tools//src/conditions:windows": [
+        "include/SDL_config.h",
+        "include/SDL_config_windows.h",
+    ],
+    "//conditions:default": [],
+})
+
+sdl_copts = select({
+    "//conditions:default": [
+        "-Wno-string-conversion",
+        "-pthread",
+        "-includestdio.h",  # Adds `#include <stdio.h>`, which some files need.
+    ],
+})
+
+sdl_linkopts = select({
+    "@bazel_tools//src/conditions:windows": [
+        "-Wl,advapi32.lib",
+        "-Wl,gdi32.lib",
+        "-Wl,imm32.lib",
+        "-Wl,ole32.lib",
+        "-Wl,oleaut32.lib",
+        "-Wl,opengl32.lib",
+        "-Wl,shell32.lib",
+        "-Wl,user32.lib",
+        "-Wl,version.lib",
+        "-Wl,winmm.lib",
+    ],
+    "//conditions:default": [
+        "-ldl",
+        "-lrt",
+    ],
+})
+
+sdl_deps = select({
+    "@bazel_tools//src/conditions:windows": [],
```

Comment:
you should be able to remove this now, right?

Reply:
Yep, done.